### PR TITLE
first capstone project 

### PR DESCRIPTION
**Pull request by tajulafreen**
In this PR, I did:

1. Make sure no linter error
2. follow design guidelines
3. personalize the content on my page
4. use bootstrap and CSS
5. responsive for both the home and about page
6. added a link to the home page when the user clicks on the logo
7.added the link to the nav bar to switch between the home and about page
8. created the mobile menu and when the user clicks it will scroll in the section view
9. added the feature speaker page dynamically in JavaScript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
...
+ - 🔰 [About Page]
+ - 🔰 [Menu Section]
+ - 🔰 [Responsive]
+-  🔰[MIT Liscence]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (brainconnect93) [inline README.md:72]: - [ ] Hello @tajulafreen, great effort working on your capstone project, But currently the project is missing the deployed link as the link in the readme file is not linking to the deployed page of your project. I would kindly suggest you deploy the project using the GitHub page and add the link to the `live demo` of your readme file.

Project Requirement 👇 
   - Live version
       - The student makes use of GitHub Pages to deploy the web pages.
       - The project is deployed properly, and all pages, links, and images work as expected.
       - A link to the deployed project is present in the README.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+
+  .hamburger {
+    visibility: hidden;
+    height: 0;
+  }
+
+  .mobile {
+    visibility: hidden;
+    height: 0;
+    width: 0;
```
Comment (brainconnect93) [inline style.css]: - [ ] Hello, I must really commend your effort so far in working on your capstone project. But there are still some things you work on in the project both on `Desktop` and `Mobile` view and below 👇 i have recorded a video guideline on what you need to pay attention to in order to make your project looks perfect and inline with what we have in the Behance original design.

Recorded Video 👇 

[screen-capture.webm](https://github.com/tajulafreen/first-capstone-project/assets/78634841/90f5d869-345b-49fc-b4d5-e5cbebecc1db)

File: index.html [hunk 3]
```diff
@@ -10,12 +10,156 @@
...
 
+<nav class="container-fluid bg-dark desktop">
+    <ul class="nav gap-4 container justify-content-end p-2">
+        <li class="nav-item"><i class="fa fa-facebook text-light"></i></li>
+        <li class="nav-item"><i class="fa fa-twitter text-light "></i></li>
+        <li class="nav-item text-light">English</li>
+        <li class="nav-item text-light">My page</li>
+        <li class="nav-item text-light">log out</li>
+    </ul>
+</nav>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you move this `nav` tag section inside the `body` of your project in the `header` tag and it should be removed from the `head` section of your project 👍 👍 

> N.B: Please implement the above changes inside the `about.html` file as well.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @tajulafreen, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight 👍

- Descriptive Pull Request title and summary. ✔️
- Project contains a Home & About page ✔️
- hamburger menu on the left side ✔️
- Follow [Cindy Shin](https://www.behance.net/adagio07) design template ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -10,12 +10,168 @@
...
```
Comment (victorokoroji) [inline index.html]: - [ ] As suggested by the previous reviewer,  I would kindly suggest you move this `nav` tag of your top navigation inside the body of your project in the `header` tag and it should be removed from the `head` section of your project  which is meant only to contain the meta tags, links and your app title👍 👍

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @tajulafreen, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (tajulafreen) [reply index.html]: Hi @vickymarz ,
Sorry, I didn't understand, what should I make changes?

File: index.html [hunk 5]
```diff
@@ -9,13 +9,169 @@
     <link rel="stylesheet" href="style.css" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
-
+</nav>
```
Comment (Ibizugbe) [inline index.html]: Nice job implementing the changes requested by the previous reviewer😉. I can see you might have mistakenly left this `nav` closing tag🥲. Kindly take it away as it makes your codebase look less professional👍.

File: index.html [hunk 6]
```diff
@@ -9,13 +9,169 @@
...
+        <hr class="lines"/>
+     </h3>
+     
+     <ul class="container part-nav">
+        <li><img src="./fries.png" alt="image" class="image"></li>
+        <li><img src="./nestle.png" alt="image" class="image"></li>
+        <li><img src="./unilever.png" alt="image" class="image"></li>
+        <li><img src="./mcdonalds.png" alt="image" class="image"></li>
+        <li><img src="./fries.png" alt="image" class="image"></li>
+     </ul>
```
Comment (Ibizugbe) [inline index.html:158]: [OPTIONAL] I suggest you always use a descriptive name for your `alt` tag as this makes it easier for screen readers to have a better idea of what the image is about👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @tajulafreen 👋👋👋, 

tryagain


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] Nice job implementing some of the changes requested by the previous reviewer👍.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @tajulafreen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**